test/run.sh

```diff
@@ -47,14 +49,20 @@ if [ $# -eq 0 ]; then
 	exit 1
 fi
 
-# load config lists
-# contains:
-#   globalTests
-#   testAlias
-#   imageTests
-#   globalExcludeTests
+# declare configuration variables
+declare -a globalTests=()
+declare -A testAlias=()
+declare -A imageTests=()
+declare -A globalExcludeTests=()
+
+# load the default config
 . "$dir/config.sh"
```

Comment:
What do you think about instead having the `--config` flag fill up the array, and if it's empty here, we put `"$dir/config.sh"` into it so that you're not forced to use our default config, but get to add it yourself if you want it? (and we don't have to add another flag to allow folks to ignore the default config if they don't want the behavior it provides)